models/tokenlon/ethereum/tokenlon_v5_ethereum_amm_v1_trades.sql

```diff
@@ -0,0 +1,116 @@
+{{ config(
+    schema = 'tokenlon_v5_ethereum',
+    tags=['dunesql'],
+    alias = alias('amm_v1_trades'),
+    partition_by = ['block_date'],
```

Comment:
moving forward, we're looking to increase our partition size to month level of granularity.
can you add a column, similar to `block_date`, that is `block_month` and use that here?

this will apply to all incremental spells in the PR.